rules.json

```diff
@@ -164,6 +164,104 @@
       }
```

Comment:
For repo maintainer, should this generated file be part of the PR or not?

Reply:
It can if it's going to be released on its own but it might cause conflicts if another release goes out before this one. Easy enough to pull in the latest and re-generate this file.